action.yml

```diff
@@ -35,6 +35,14 @@ inputs:
     description: "Port for JSON-RPC-Relay"
     required: false
     default: "7546"
+  grpcProxyPort:
+    description: "Port for gRPC Proxy"
+    required: false
+    default: "9998"
+  haproxyPort:
+    description: "Port for HAProxy"
+    required: false
+    default: "50211"
```

Comment:
Great PR @gsstoykov. Could we add a step to `Validate HAProxy and gRPC Proxy` in the workflows to confirm this works fine.